app/src/main/kotlin/com/intive/picover/parties/repository/PartiesRepository.kt

```diff
@@ -20,6 +21,15 @@ class PartiesRepository @Inject constructor(
 			}
 		}
 
+	suspend fun addParty(partyRemote: PartyRemote) =
+		runCatching {
+			partiesReference.push().let { databaseReference ->
+				databaseReference.key?.let { key ->
```

Comment:
so what will happen now if the key is null ? just nothing will happen ? shouldn't we force the key here ?  is it possible to have no key here ?

Reply:
Ok, I will try to reproduce that tomorrow morning

Reply:
There is nothing to reproduce, just think about consequences :)

Reply:
@kmaslowiec Perfectly, that's the point and right conclusion :)

Reply:
@kmaslowiec that's the spirit :D

Reply:
Updated. I was wondering if I should throw more specific error such as IllegalStateExcepption but I think a general error message 'Something went wrong" should be enough for such a corner case.